components/dashboard/src/usage/UsageView.tsx

```diff
@@ -96,26 +89,25 @@ export const UsageView: FC<UsageViewProps> = ({ attributionId }) => {
 
     const usageEntries = usagePage.data?.usageEntriesList || [];
 
-    const schedulerDuration = useFeatureFlag("usage_update_scheduler_duration");
-
     const readableSchedulerDuration = useMemo(() => {
-        const duration = schedulerDuration.toString().toLowerCase();
-        if (duration === "undefined") {
-            return "15 minutes";
+        const intervalMinutes = usagePage.data?.ledgerIntervalMinutes;
+        if (!intervalMinutes) {
+            return "";
         }
-        const unit = duration.slice(-1);
-        const unitStr = durationUnitMap[unit];
-        if (!unitStr) {
-            console.error("failed to parse duration", duration);
-            return "15 minutes";
-        }
-        const value = parseInt(duration.slice(0, -1), 10);
-        return `${value} ${unitStr}`;
-    }, [schedulerDuration]);
+
+        return `${intervalMinutes} minutes`;
```

Comment:
It looks like the default is one minute, at least on the preview environment, so this is so that we format that case correctly.

```suggestion
        return `${intervalMinutes} minute${intervalMinutes !== 1 ? "s" : "" }`;
```